apps/financial-remedy/finrem-cos/aat-image-policy.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^pr-859-[a-f0-9]+-(?P<ts>[0-9]+)'
+    pattern: '^pr-2246-[a-f0-9]+-(?P<ts>[0-9]+)'
```

Comment:
that's really weird. why is there a pr on there already? if you think about it... it can't be using it